### ISSUE TITLE
LWVol_BfTier1 and LWVol_WetTier don't show up in Metric Review graph

### ISSUE THREAD

**Issue by volkcj** [SouthForkResearch/CHaMP_Metrics#92]
Schema is QA-Aux Tier 1 and these metrics appear in the table of the Metric Review view, but they don't show up in the graph.
![2017-10-11_111500](https://user-images.githubusercontent.com/13526689/31458647-8d509796-ae75-11e7-83ad-4932ae32e2e6.png)

**Comment by volkcj** [comment]
This actually doesn't work for TopoTier1 too, so this might be how it's supposed to work. I can't remember if we passed on the graphing capabilities for non-Visit level schemas.

**Comment by philipbaileynar** [comment]
Correct. This graph was discussed but too involved to implement quickly in 2017. This is correctly labeled as an enhancement.
